include/Gaffer/SplinePlug.h

```diff
@@ -124,14 +132,17 @@ class SplinePlug : public ValuePlug
 		YPlugType *pointYPlug( unsigned pointIndex );
 		const YPlugType *pointYPlug( unsigned pointIndex ) const;
 
-		IntPlug *endPointMultiplicityPlug();
-		const IntPlug *endPointMultiplicityPlug() const;
+		// If you are starting with a curve representation that needs duplicated end point values, and you're
+		// converting it into this plug representation, you should trim off the duplicated end point values first
+		static bool trimEndPoints( typename T::PointContainer &points, Interpolation interpolation );
```

Comment:
Do we need this anywhere apart from the shader loaders? If not, I'd be tempted to squirrel it away as a protected method on the Shader base class.